docs/pipelines.md

```diff
@@ -369,7 +371,7 @@ of the `TaskRun` that executes that `Task` within the `PipelineRun` that execute
 The `Timeout` value is a `duration` conforming to Go's [`ParseDuration`](https://golang.org/pkg/time/#ParseDuration)
 format. For example, valid values are `1h30m`, `1h`, `1m`, and `60s`. 
 
-**Note:** If you do not specify a `Timeout` value, Tekton instead honors the timeout for the [`PipelineRun`](pipelineruns.md#syntax).
+**Note:** If you do not specify a `Timeout` value, Tekton instead honors the timeout for the [`PipelineRun`](pipelineruns.md#configuring-a-pipelinerun).
```

Comment:
I can't find a heading in `pipelineruns.md` that correlates to this fragment ID (`configuring-a-pipelinerun`).  Could you point me at the spot this is linking to in our docs?

Reply:
This link will work once we submit https://github.com/tektoncd/pipeline/pull/2364, so maybe we can just submit this PR when we submit 2364?

Reply:
ah i see, ok!